### ISSUE TITLE
Fails to parse statement where prefixed name with no local name immediately followed by dot

### ISSUE THREAD

**Issue by pduchesne** [RDFLib/rdflib#1664]
The following turtle sample fails to be parsed, because of the lack of whitespace between `object:` and the terminal dot: 
```
@prefix test: <http://example.org/>.
@prefix object: <http://example.org/object>.
test:subject test:predicate object:.
```

In (my understanding of) the Turtle grammar, this is valid.
Turtle validator such as http://ttl.summerofcode.be concurs.

To the question : "why such a construct", the turtle serializer of [linkeddata/rdflib.js ](https://github.com/linkeddata/rdflib.js/) f.i. does produce this.

**Comment by nicholascar** [comment]
@niklasl yet another Turtle niggle...

**Comment by niklasl** [comment]
Yes, this is valid Turtle. But it appears that recent versions of RDFLib does handle this correctly. @pduchesne which version of RDFLib do you use? The following code works with RDFLib 6.1.1:
```python
import rdflib

g = rdflib.Graph().parse(data= """
    @prefix test: <http://example.org/>.
    @prefix object: <http://example.org/object>.
    test:subject test:predicate object:.
    """, format="text/turtle")

assert len(g) == 1
print(g.serialize(format="text/turtle"))
```

**Comment by aucampia** [comment]
Confirming this works with #1677

**Comment by nicholascar** [comment]
OK, this will be closed with the merger of #1667
